src/Umbraco.Cms.Api.Management/Controllers/AuditLog/CurrentUserAuditLogController.cs

```diff
@@ -17,28 +18,24 @@ public class CurrentUserAuditLogController : AuditLogControllerBase
 {
     private readonly IAuditService _auditService;
     private readonly IAuditLogPresentationFactory _auditLogPresentationFactory;
-    private readonly IUserService _userService;
+    private readonly IBackOfficeSecurityAccessor _backOfficeSecurityAccessor;
 
     public CurrentUserAuditLogController(
         IAuditService auditService,
         IAuditLogPresentationFactory auditLogPresentationFactory,
-        IUserService userService)
+        IBackOfficeSecurityAccessor backOfficeSecurityAccessor)
     {
         _auditService = auditService;
         _auditLogPresentationFactory = auditLogPresentationFactory;
-        _userService = userService;
+        _backOfficeSecurityAccessor = backOfficeSecurityAccessor;
     }
 
     [HttpGet]
     [MapToApiVersion("1.0")]
     [ProducesResponseType(typeof(PagedViewModel<AuditLogWithUsernameResponseModel>), StatusCodes.Status200OK)]
     public async Task<IActionResult> CurrentUser(Direction orderDirection = Direction.Descending, DateTime? sinceDate = null, int skip = 0, int take = 100)
     {
-        // FIXME: Pull out current backoffice user when its implemented.
-        // var userId = _backOfficeSecurityAccessor.BackOfficeSecurity?.GetUserId().Result ?? -1;
-        var userId = Constants.Security.SuperUserId;
-
-        IUser? user = _userService.GetUserById(userId);
+        IUser? user = _backOfficeSecurityAccessor.BackOfficeSecurity?.CurrentUser;
```

Comment:
Use `CurrentUser(_backOfficeSecurityAccessor)` (implemented on `ManagementApiControllerBase`) and remove the exception throwing below 😄